src/app/visit/visit.component.ts

```diff
@@ -147,31 +151,95 @@ export class VisitComponent implements OnInit {
     pageTriggerManually(): void {
         this.paginatorPageChecker();
     }
-
     sendVisit(): void {
-        const visit: Visit = {
-            patientId: this.patient._id,
-            note: this.noteVal,
+        // const visit: Visit = {
+        //     patientId: this.patient._id,
+        //     note: this.noteVal,
+        //     points: this.visitPoints,
+        // };
+        const pdfData = {
+            fullName: this.patient.name + ' ' + this.patient.surname,
+            logo: this.company.logo,
+            generickInfo: this.company.genericInfo,
             points: this.visitPoints,
         };
-        this.visitService.createVisit(visit).subscribe({
-            next: (data) => {
-                this.snackService.showSnackBarMessage(
-                    'SUCCESS.PATIENT_VISIT_CREATE_VISIT',
-                    SNACK_TYPE.success,
-                );
-                console.log(data);
+        const docDefinition = {
+            content: [
+                {
+                    text: this.datePicker(),
```

Comment:
use a angular [date pipe](https://www.angularjswiki.com/angular/how-to-use-angular-pipes-in-components-and-services/)